xclim/data/fr.json

```
@@ -733,42 +732,42 @@
+    "abstract": "Calcul de l'humidité relative calculée à partir de la température et du point de rosée à l'aide de la pression de vapeur saturante."
   },
   "HURS": {
     "long_name": "Humidité relative",
     "description": "Calculée à partir de la température, de l'humidité spécifique et de la pression à l'aide de la pression de vapeur saturante, laquelle fut calculée en suivant la méthode {method}.",
-    "title": "Humidité relative à partir de l'humidité spécifique et de la pression",
-    "abstract": "Calcul de l'humidité relative à partir de la température, de l'humidité spécifique et de la pression à l'aide de la pression de vapeur saturante."
+    "title": "Humidité relative calculée à partir de la température, l'humidité spécifique et de la pression",
+    "abstract": "Calcul de l'humidité relative calculée à partir de la température, de l'humidité spécifique et de la pression à l'aide de la pression de vapeur saturante."
   },
   "HUSS": {
     "long_name": "Humidité spécifique",
     "description": "Calculée à partir de la température, de l'humidité relative et de la pression à l'aide de la pression de vapeur saturante, laquelle fut calculée en suivant la méthode {method}.",
-    "title": "Humidité spécifique à partir de l'humidité relative et de la pression",
-    "abstract": "Calcul de l'humidité spécifique à partir de la température, de l'humidité relative et de la pression à l'aide de la pression de vapeur saturante."
+    "title": "Humidité spécifique calculée à partir de la température, l'humidité relative et de la pression",
+    "abstract": "Calcul de l'humidité spécifique calculée à partir de la température, de l'humidité relative et de la pression à l'aide de la pression de vapeur saturante."
   },
   "HUSS_FROMDEWPOINT": {
     "long_name": "Humidité spécifique",
     "description": "Calculée à partir de la température du point de rosée et de la pression à l'aide de la pression de vapeur saturante, laquelle fut calculée en suivant la méthode {method}.",
-    "title": "Humidité spécifique à partir de la température du point de rosée et de la pression",
-    "abstract": "Calcul de l'humidité spécifique à partir de la température du point de rosée et de la pression à l'aide de la pression de vapeur saturante."
+    "title": "Humidité spécifique calculée à partir de la température du point de rosée et de la pression",
+    "abstract": "Calcul de l'humidité spécifique calculée à partir de la température du point de rosée et de la pression à l'aide de la pression de vapeur saturante."
   },
   "FIRST_DAY_BELOW": {
-    "long_name": "Premier jour de l'année avec une température sous {thresh}",
-    "description": "Premier jour de l'année avec une température sous {thresh} pour au moins {window} jours.",
+    "long_name": "Premier jour de l'année avec une température sous un seuil",
+    "description": "Premier jour de l'année avec une température sous {thresh} pour au moins {window} jours après {after_date}.",
```

```suggestion
    "description": "Premier jour de l'année après {after_date} ayant une température sous {thresh} pour au moins {window} jours .",
```
Sinon, on dirait que le window réfère au nombre de jour après after_date.